.eslintrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  parser: "babel-eslint",
```

Comment:
This was already installed but not set in the root ESLint config. Setting this allowed ESLint to parse the export-default-from syntax.